packages/db-base/src/lib/inMemFileDB.ts

```diff
@@ -191,21 +254,27 @@ class InMemoryFileDB {
         this.settings.backup.period *= 60000;
 
         this.settings.backup.files =
-            this.settings.backup.files === undefined ? 24 : parseInt(this.settings.backup.files);
+            this.settings.backup.files === undefined ? 24 : parseInt(String(this.settings.backup.files));
         if (isNaN(this.settings.backup.files)) {
             this.settings.backup.files = 24;
         }
 
         this.settings.backup.hours =
-            this.settings.backup.hours === undefined ? 48 : parseInt(this.settings.backup.hours);
+            this.settings.backup.hours === undefined ? 48 : parseInt(String(this.settings.backup.hours));
         if (isNaN(this.settings.backup.hours)) {
             this.settings.backup.hours = 48;
         }
         // Create backup directory
         fs.ensureDirSync(this.backupDir);
     }
 
-    handleSubscribe(client, type, pattern, options, cb) {
+    handleSubscribe(
+        client: SubscriptionClient,
+        type: string,
+        pattern: string | string[],
+        options: any,
+        cb: () => void
+    ): void {
         if (typeof options === 'function') {
             cb = options;
             options = undefined;
```

Comment:
We really need to define the possible overloads so this stuff can be handled at the call sites too. I can help with that once the other changes are complete.